src/main/kotlin/platform/mixin/inspection/shadow/ShadowModifiersInspection.kt

```diff
@@ -103,6 +103,9 @@ class ShadowModifiersInspection : MixinInspection() {
             // TODO: Would it make sense to apply the @Final check to methods?
             if (member !is PsiField) {
                 return
+            } else if (member.hasInitializer()) {
```

Comment:
Nit: could you change this from an `else if` to a regular `if`?

Reply:
I'll check if type inference still holds in that situation - I want to avoid an ugly cast to `PsiField` and I figured this was the easiest way to do it.

Reply:
It should work because of the return